### PR TITLE
US5 delete recipe stephan

### PR DESCRIPTION
**Pull request by StephanLippert**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "eslint": "8.44.0",
     "eslint-config-next": "13.4.8",
+    "modal": "^1.2.0",
```
Comment (onemanwenttomow) [inline package.json:15]: Did you mean to install [this](https://www.npmjs.com/package/modal)?

You can uninstall it by running `npm uninstall modal` in the terminal

File: components/Modal/ModalStyle.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  box-shadow: 0 2px 2px rgb(5, 133, 14);
+`;
+export const StyledButtonNo = styled.button`
+  margin: 8px;
+  padding: 8px 16px;
+  border: solid 2px rgb(153, 9, 45);
+  cursor: pointer;
+  border-radius: 5px;
+  box-shadow: 0 2px 2px rgb(153, 9, 45);
+`;
```
Comment (onemanwenttomow) [inline components/Modal/ModalStyle.js:44]: There seems to be a fair amount of repeated code here. Maybe it makes sense to have a base button, and then the delete, yes and no buttons extend from that. [See docs here](https://styled-components.com/docs/basics#extending-styles).

e.g.

``` js
const Button = styled.button`
  margin: 8px;
  padding: 8px 16px;
  cursor: pointer;
  border-radius: 5px;
`
export const StyledDeleteButton = styled(Button)`
  border: solid 2px rgb(153, 9, 45);
  box-shadow: 0 2px 2px rgb(153, 9, 45);
`;

export const StyledButtonYes = styled(Button)`
  border: solid 2px rgb(5, 133, 14);
  box-shadow: 0 2px 2px rgb(5, 133, 14);
`;

export const StyledButtonNo = styled(Button)`
  border: solid 2px rgb(153, 9, 45);
  box-shadow: 0 2px 2px rgb(153, 9, 45);
`;

```

Even here `StyledButtonNo` and `StyledDeleteButton` seem to be having the same styling. Maybe it makes sense to just have one component?

File: pages/recipes/[id]/index.js [hunk 3]
```diff
@@ -6,13 +8,27 @@ import {
...
 
 export default function DetailsPage() {
+  const [showModal, setShowModal] = useState(false);
   const router = useRouter();
   const { isReady } = router;
   const { id } = router.query;
   const { data: recipe, isLoading, error } = useSWR(`/api/recipes/${id}`);
+  async function deleteRecipe() {
+    await fetch(`/api/recipes/${id}`, {
+      method: "DELETE",
```
Comment (onemanwenttomow) [inline pages/recipes/[id]/index.js:27]: You don't seem to be checking for this request method in your API route. 

Also, i don't see where you are deleting the item from the database.

**Review by onemanwenttomow — CHANGES_REQUESTED** [review]
Hello. First of all great job so far. 

I do think that there is a major part missing (i.e. the API route and the deletion from the database).

File: components/Modal/ModalStyle.js (hunk 2 above)
Comment (StephanLippert) [reply components/Modal/ModalStyle.js:44]: Thanks for the suggestion. Our plan is to make a hole Style-US a bit later.

File: package.json (hunk 1 above)
Comment (StephanLippert) [reply package.json:15]: we uninstalled it. But as it still works we are not sure if we got your point right ;-)

Comment (onemanwenttomow) [reply package.json]: I don't think you were using it in your project. So the fact that you uninstalled it and everything still works is a good sign.

**Review by NikRoe — APPROVED** [review]
LGTM 👌

**Pull request merged**